.github/workflows/ci.yml

```diff
@@ -356,6 +356,7 @@ jobs:
             !:trino-iceberg,
             !:trino-ignite,
             !:trino-jdbc,
+            !:trino-jdbc-thin,
```

Comment:
We might want to pick a different name. “Thin” has a special meaning in the world of jdbc drivers (pure Java, no dependencies needed, easy to install and set up), and it’s almost the opposite of what this driver provides.

Reply:
thin vs fat - that's my understanding

Reply:
What name do you propose?

Reply:
maybe "slim"?

Reply:
maybe core or slim is also fine by me

Reply:
trino-jdbc-core seems nice